src/cloud-element-templates/cmd/ChangeElementTemplateHandler.js

```diff
@@ -900,6 +897,46 @@ export default class ChangeElementTemplateHandler {
     return bo && bo.get('messageRef');
   }
 
+  _ensureMessage(element, template) {
+
+    const message = this._getMessage(element);
+
+    // message is already templated, so we use it
+    if (message && message.get('zeebe:modelerTemplate')) {
+      return message;
+    }
+
+    const newMessage = this._createMessage(element, template);
+
+    // no message is set on the element, so no properties to copy
+    if (!message) {
+      return newMessage;
+    }
+
+    // copy name property
```

Comment:
Why don't we `moddleCopy` the message?
https://github.com/bpmn-io/bpmn-js/blob/9a67633ead8c196dc055d917c67b277137ecb7fb/test/spec/features/copy-paste/ModdleCopySpec.js#L50

Reply:
Perhaps it could also make sense to have a test to ensure that generated values are still re-generated: https://github.com/bpmn-io/bpmn-js-element-templates/blob/059645e9fc1fa29bfb6f2d104312dadc807cfd84/test/spec/cloud-element-templates/cmd/generated-values.json#L74